File: core/src/main/java/cz/dynawest/jtexy/parsers/AroundEvent.java

package cz.dynawest.jtexy.parsers;

import cz.dynawest.jtexy.modules.TexyModifier;

/**
 *
 * @author Ondrej Zizka
 */
public abstract class AroundEvent extends TexyParserModEvent {

	public AroundEvent(TexyParser parser, String text, TexyModifier modifier) {
		super(parser, text, modifier);
	}

}// class

File: core/src/main/java/cz/dynawest/jtexy/parsers/AroundEvent.kt
package cz.dynawest.jtexy.parsers

import cz.dynawest.jtexy.modules.TexyModifier

/**
 *
 * @author Ondrej Zizka
 */
abstract class AroundEvent(parser: TexyParser?, text: String?, modifier: TexyModifier?) : TexyParserModEvent(parser, text, modifier)
